app/views/activities/edit.html.slim

```diff
@@ -0,0 +1,10 @@
+p
+  | Edit activity
+= form_for @activity do |f|
```

Comment:
I would suggest to extract editing form to _form.html.slim in activities folder and reuse it for both - new and edit action.

You can idea about that via making dummy rails app and make a scaffold in it to some resource